docs/StardustDocs/topics/installation.md

```diff
@@ -52,45 +52,57 @@ The simplest way of doing this is shown on screenshot:
 The Kotlin DataFrame library is published to Maven Central, so you can simply add the following line to your Kotlin DSL
 buildscript to depend on it:
 
-### All-in-one artifact
+### General configuration
 
 <tabs>
 <tab title="Kotlin DSL">
 
 ```kotlin
+plugins {
+    id("org.jetbrains.kotlinx.dataframe") version "%dataFrameVersion%"
+}
+
 dependencies {
-    implementation("org.jetbrains.kotlinx:dataframe:<version>")
+    implementation("org.jetbrains.kotlinx:dataframe:%dataFrameVersion%")
 }
 ```
 
 </tab>
 
 <tab title="Groovy DSL">
 
-```kotlin
+```groovy
+plugins {
```

Comment:
is it ok with formatting, I could not understand?

Reply:
I think yes, looks similar to the code generated by a project wizzard